script/webots-firejail

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Run webots controllers in jails
+export WEBOTS_FIREJAIL_CONTROLLERS=1
+
+REPO_ROOT=$(dirname $0)/..
+
+# We want the Firejailed process to have access to the siblings of the
+# simulation directory because the  competitors code lives in sibling directories.
```

Comment:
Double space and grammar. What idiot wrote this?
```suggestion
# simulation directory because the competitors' code lives in sibling directories.
```